### DISCUSSION TITLE
Anonymous pull for images in ghcr fails

### DISCUSSION THREAD

**Discussion by rtate-tunein** [community/community#27045] [Actions]
<p>Actions are currently failing with the error message:</p>
<pre><code class="lang-auto"> /usr/bin/docker pull ghcr.io/rtcamp/action-slack-notify:v2.1.3
  v2.1.3: Pulling from rtcamp/action-slack-notify
  c9b1b535fdd9: Pulling fs layer
  f597b81b5d49: Pulling fs layer
  5315175e77ba: Pulling fs layer
  180cd15dbe12: Pulling fs layer
  f47731e8b606: Pulling fs layer
  35c5edac0302: Pulling fs layer
  25c20a50cc35: Pulling fs layer
  f47731e8b606: Waiting
  35c5edac0302: Waiting
  25c20a50cc35: Waiting
  180cd15dbe12: Waiting
  error pulling image configuration: denied: unauthenticated: User cannot be authenticated with the token provided.
</code></pre>
<p>I don’t think this issue is just related to actions however. I cannot <code>docker pull</code> any public ghcr image found in <a href="https://github.com/search?package_type=container&amp;q=alpine&amp;type=RegistryPackages" rel="noopener nofollow ugc">this search</a> for instance. Same error message:</p>
<pre><code class="lang-auto">$ docker pull ghcr.io/gjchentw/alpine:edge
edge: Pulling from gjchentw/alpine
fa7045767063: Pulling fs layer
a1a95791ff2c: Pulling fs layer
426e3ad18ba7: Pulling fs layer
fe0e350e885c: Waiting
error pulling image configuration: denied: unauthenticated: User cannot be authenticated with the token provided.
</code></pre>

**Comment by lirantal** [comment, marked as answer]
<p><a class="mention" href="/rtate-tunein">@rtate-tunein</a>  :wave:</p>
<p>Perhaps this was an intermittent networking or internal issue on the GitHub side?<br>
I have successfully pulled both of these images now.</p>

**Comment by rtate-tunein** [comment]
<p>I believe that’s correct. Later in the day GHCR started working again</p>

**Comment by lirantal** [comment]
<p>Thanks for confirming!</p>
